imxrt-hal/src/gpio.rs

```diff
@@ -236,7 +236,19 @@ ios! {
     17, GPIO1IO17: [GPIO1, gpio17, crate::iomuxc::gpio::GPIO_AD_B1_01<crate::iomuxc::Alt5>, FAST: (GPIO6, gpr26)],
     18, GPIO1IO18: [GPIO1, gpio18, crate::iomuxc::gpio::GPIO_AD_B1_02<crate::iomuxc::Alt5>, FAST: (GPIO6, gpr26)],
     19, GPIO1IO19: [GPIO1, gpio19, crate::iomuxc::gpio::GPIO_AD_B1_03<crate::iomuxc::Alt5>, FAST: (GPIO6, gpr26)],
-    // TODO: more from this bank? Missing 20..=31
+    20, GPIO1IO20: [GPIO1, gpio20, crate::iomuxc::gpio::GPIO_AD_B1_04<crate::iomuxc::Alt5>, FAST: (GPIO6, gpr26)],
+    21, GPIO1IO21: [GPIO1, gpio21, crate::iomuxc::gpio::GPIO_AD_B1_05<crate::iomuxc::Alt5>, FAST: (GPIO6, gpr26)],
+    21, GPIO1IO22: [GPIO1, gpio22, crate::iomuxc::gpio::GPIO_AD_B1_06<crate::iomuxc::Alt5>, FAST: (GPIO6, gpr26)],
+    21, GPIO1IO23: [GPIO1, gpio23, crate::iomuxc::gpio::GPIO_AD_B1_07<crate::iomuxc::Alt5>, FAST: (GPIO6, gpr26)],
+    21, GPIO1IO24: [GPIO1, gpio24, crate::iomuxc::gpio::GPIO_AD_B1_08<crate::iomuxc::Alt5>, FAST: (GPIO6, gpr26)],
+    21, GPIO1IO25: [GPIO1, gpio25, crate::iomuxc::gpio::GPIO_AD_B1_09<crate::iomuxc::Alt5>, FAST: (GPIO6, gpr26)],
+    21, GPIO1IO26: [GPIO1, gpio26, crate::iomuxc::gpio::GPIO_AD_B1_10<crate::iomuxc::Alt5>, FAST: (GPIO6, gpr26)],
+    21, GPIO1IO27: [GPIO1, gpio27, crate::iomuxc::gpio::GPIO_AD_B1_11<crate::iomuxc::Alt5>, FAST: (GPIO6, gpr26)],
+    21, GPIO1IO28: [GPIO1, gpio28, crate::iomuxc::gpio::GPIO_AD_B1_12<crate::iomuxc::Alt5>, FAST: (GPIO6, gpr26)],
+    21, GPIO1IO29: [GPIO1, gpio29, crate::iomuxc::gpio::GPIO_AD_B1_13<crate::iomuxc::Alt5>, FAST: (GPIO6, gpr26)],
+    21, GPIO1IO30: [GPIO1, gpio30, crate::iomuxc::gpio::GPIO_AD_B1_14<crate::iomuxc::Alt5>, FAST: (GPIO6, gpr26)],
+    21, GPIO1IO31: [GPIO1, gpio31, crate::iomuxc::gpio::GPIO_AD_B1_15<crate::iomuxc::Alt5>, FAST: (GPIO6, gpr26)],
```

Comment:
The first argument for each line, `offset`, in this selected code should be `22, 23, 24, 25, ..., 31`. They're all `21` right now.

The mapping between GPIO bank to chip pad (`GPIO1_22 => GPIO_AD_B1_06` mapping) for the newly-added code looks good!

Reply:
Doh, thanks!